src/status.js

```diff
@@ -0,0 +1,8 @@
+export function updateCompletedStatus(task, items) {
+  const index = items.findIndex((item) => item.index === task.index);
+  items[index].completed = !items[index].completed;
+}
+
+export function clearCompleted(items) {
+  return items.filter((task) => !task.completed);
+}
```

Comment:
- [ ] [OPTIONAL] These functionalities of the app are not part of the current milestone. You'll have the chance to work on them in the next milestone. You can remove them for now. :+1: